src/main/java/levi9/VehicleService/service/impl/AddressServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package levi9.VehicleService.service.impl;
+
+import levi9.VehicleService.dto.AddressDto;
+import levi9.VehicleService.repository.AddressRepository;
+import levi9.VehicleService.service.AddressService;
+import lombok.AllArgsConstructor;
+import org.modelmapper.ModelMapper;
+import org.modelmapper.TypeToken;
+import org.springframework.stereotype.Service;
+
+import java.lang.reflect.Type;
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+@AllArgsConstructor
+public class AddressServiceImpl implements AddressService {
+    private final AddressRepository addressRepository;
+    private final ModelMapper mapper = new ModelMapper();
```

Comment:
This mapper can be added as a @Bean in some @Configuration class, and then autowired here and on other places where it is used as well.